spec/factories/user_programme_enrolments.rb

```diff
@@ -3,10 +3,17 @@
     user
     programme
 
+    before(:create) do |enrolment|
+      if enrolment.programme.pathways? && enrolment.pathway.nil?
```

Comment:
If a pathway hasn't been explicitly picked when creating an enrolment to primary or secondary, we pick the first pathway form those programmes